tests/pkdebug1_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
this should be pkdebug2_test.py. The other should be renamed pkdebug1_test.py.  It's a bit confusing having pkdebug_test.py and pkdebug1_test.py, since that's not done elsewhere.